lib/jxl/jpeg/enc_jpeg_data_reader.cc

```diff
@@ -23,50 +23,42 @@
 #define JXL_DEBUG_JPEG_DATA_READER JXL_DEBUG_ON_ERROR
 #endif  // JXL_DEBUG_JPEG_DATA_READER
 
-#define JXL_JPEG_DEBUG(format, ...) \
-  JXL_DEBUG(JXL_DEBUG_JPEG_DATA_READER, format, ##__VA_ARGS__)
+#define JXL_JPEG_FAILURE(format, ...)                           \
+  JXL_DEBUG(JXL_DEBUG_JPEG_DATA_READER, format, ##__VA_ARGS__); \
```

Comment:
Ah yes, that is indeed better. Just changing it to `return JXL_FAILURE` at this point, because there isn't much reason to define another macro for something that already exists.